scraper/src/github.py

```diff
@@ -198,10 +200,83 @@ def add_collaborations(self, event, event_time):
                         "time": event_time,
                         "link": event["payload"]["pull_request"]["html_url"],
                         "text": event["payload"]["pull_request"]["title"],
-                        "collaborated_with": list(others)
+                        "collaborated_with": list(others),
                     },
                 )
 
+    def caclculate_turnaround_time(self, event):
+        user = event["payload"]["pull_request"]["user"]["login"]
+        merged_at = datetime.strptime(
+            event["payload"]["pull_request"]["merged_at"], "%Y-%m-%dT%H:%M:%SZ"
+        )
+        created_at = datetime.strptime(
+            event["payload"]["pull_request"]["created_at"], "%Y-%m-%dT%H:%M:%SZ"
+        )
+
+        linked_issues = re.findall(
+            r"(fix|fixes|fixed|close|closes|closed|resolve|resolves|resolved) ([\w\/.-]*)(#\d+)",
+            event["payload"]["pull_request"]["body"] or "",
+            re.IGNORECASE | re.MULTILINE,
+        )
```

Comment:
This wouldn't capture issues that are linked through the UI right?

<img width="543" alt="image" src="https://github.com/coronasafe/leaderboard/assets/25143503/63e5e010-8427-4b1d-b281-41b5ab3e35a9">